src/giskardpy/utils.py

```diff
@@ -165,6 +166,36 @@ def cylinder_surface(r, h):
 #             return True
 #     return False
 
+def calculate_waypoint2D(target, origin, distance):
+    """
+    Calculates a waypoint in front of the target.
+    :param target: The target position
+    :type target: Quaternion
```

Comment:
That should be ```Point``` no?

Reply:
Actually it is Point, but the Docstring is incorrect. It is Changed, but not displayed here